modules/spring-config/src/main/resources/META-INF/mule-module.xsd

```diff
@@ -150,24 +150,42 @@
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
-                <xsd:attribute name="namespace" use="required">
+                <xsd:attribute name="category" type="categoryType" default="COMMUNITY">
                     <xsd:annotation>
                         <xsd:documentation>
-                            Expected namespace of the module to look for when generating the schemas.
+                            Expected category of the module.
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
-                <xsd:attribute name="category" type="categoryType" default="COMMUNITY">
+                <xsd:attribute name="vendor" type="xsd:string" default="MuleSoft">
                     <xsd:annotation>
                         <xsd:documentation>
-                            Expected category of the module.
+                            Expected vendor of the module.
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
-                <xsd:attribute name="vendor" type="xsd:string" default="MuleSoft">
+                <xsd:attribute name="minMuleVersion" type="xsd:string" use="optional" default="4.0.0">
                     <xsd:annotation>
                         <xsd:documentation>
-                            Expected vendor of the module.
+                            Min Mule version that the extension requires to work correctly.
+                        </xsd:documentation>
+                    </xsd:annotation>
+                </xsd:attribute>
+                <!-- DSL XML attributes -->
+                <xsd:attribute name="namespace" use="optional">
+                    <xsd:annotation>
+                        <xsd:documentation>
+                            Expected namespace of the module to look for when generating the schemas. If left empty it will
+                            default to http://www.mulesoft.org/schema/mule/[prefix], where [prefix] is the attribute defined below.
+                        </xsd:documentation>
+                    </xsd:annotation>
+                </xsd:attribute>
+                <xsd:attribute name="prefix" use="optional">
```

Comment:
This is inconsistent with the default behaviour in java based connectors (where the concept of prefix doesn't exist at all). Why should smart connector have a different way of configuring this? Why don't they configure the same stuff as in the @Xml annotation?